cli/tests/test_rebase_command.rs

```diff
@@ -496,6 +498,32 @@ fn test_rebase_multiple_destinations() {
     ◉
     "###);
 
+    // undo and do it again, but with 'revsets.always-allow-large-revsets'
+    let (stdout, stderr) = test_env.jj_cmd_ok(&repo_path, &["undo"]);
+    insta::assert_snapshot!(stdout, @"");
+    insta::assert_snapshot!(stderr, @"");
+    let (stdout, stderr) = test_env.jj_cmd_ok(
+        &repo_path,
+        &[
+            "rebase",
+            "--config-toml=revsets.always-allow-large-revsets=true",
+            "-r",
+            "a",
+            "-d",
+            "b|c",
+        ],
+    );
+    insta::assert_snapshot!(stdout, @"");
+    insta::assert_snapshot!(stderr, @"");
```

Comment:
i'm not sure we need more coverage of the output of `jj rebase` here, but at least this is in `test_rebase_command.rs`, so it makes some sense. up to you

Reply:
It was basically just a copy/paste job, but I think removing it makes sense.